api/client/client.go

```diff
@@ -3217,6 +3218,7 @@ func (c *Client) GetApp(ctx context.Context, name string) (types.Application, er
 //
 // For a full list of registered applications that are served by an application
 // service, use GetApplicationServers instead.
+// Deprecated: Prefer using [ListApps] or [Apps] instead.
```

Comment:
I don't know if we should deprecate methods in the api module in the middle of a major version without good reason - this might be a good reason, however. 🤔  

```suggestion
```